CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # legacysurvey Change Log
 
+
+## 8.0.1 (DR7, 2018-07-26)
```

Comment:
Cut & paste from DR7

Reply:
Thanks for looking at this @weaverba137. Let's update the parenthetical and resolve this conversation whenever @stephjuneau replies and the PR gets merged. I don't want to change this today and then have the PR be merged into master next week...

Reply:
That's fine.

Reply:
Hi @geordie666 & @weaverba137, to follow up from my reply below, please let me know if you have an opinion about serving the separate dr8-north and dr8-south collections, just the combined dr8 collection, or a third option which would be to serve 3 collections with the separated footprints, and the combined one.

> Sorry for being late to comment here. So the main difference from dr7 is that I thought we would serve the two ("north" and "south") footprints separately, which means that there would be two collections: "Legacy Survey: DR8-north" and "Legacy Survey: DR8-south", which I think is the most similar to how the files are served at NERSC? Unless you have an objection and want us to combine into only one collection? A closer examination also revealed another discrepancy, as written in the attached screenshot. I will follow with Sean McManus, who said he expects to work on this during this week.
> ![LS_noao_screenshot](https://user-images.githubusercontent.com/26126576/61754187-ffb78f00-ad66-11e9-94f4-aa5dcdb1134e.jpg)